src/state/connection-mixin.ts

```diff
@@ -97,6 +91,12 @@ export const connectionMixin = <T extends Constructor<HassBaseEl>>(
                 err
               );
             }
+            if (
+              err.error.code === ERR_CONNECTION_LOST &&
```

Comment:
```suggestion
              err.error?.code === ERR_CONNECTION_LOST &&
```
Otherwise it causes frontend exceptions, when a backend service raises an `HomeAssistantError`.

Reply:
fixed